modules/utxo-lib/src/bitgo/wallet/Unspent.ts

```diff
@@ -1,14 +1,15 @@
 import { Network } from '../..';
```

Comment:
Should this file be `WalletUnspent.ts`? It seems like it'd go along with the naming convention for the other files within this folder, and would distinguish it from the `Unspent.ts` in the parent dir.

Reply:
it probably should